src/App.tsx

```diff
@@ -34,8 +34,8 @@ import { TransactionWatcherProvider } from 'contexts/TransactionWatcher'
 import { V3FarmingProvider } from 'contexts/V3Farming'
 import { BedTokenMarketDataProvider } from 'contexts/BedTokenMarketData'
 import { BedIndexComponentsProvider } from 'contexts/BedIndexComponents'
-import { StreamingFeeProvider } from "contexts/StreamingFee"
-import { TokenSupplyProvider } from "contexts/TokenSupply"
+import { StreamingFeeProvider } from 'contexts/StreamingFee'
+import { TokenSupplyProvider } from 'contexts/TokenSupply'
```

Comment:
heads up small merge conflict: https://github.com/SetProtocol/index-ui/pull/502 which does this across the codebase

Reply:
Sounds good! I'll rebase after merging #502

Reply:
glhf